custom-sign-in/App.js

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019, Okta, Inc. and/or its affiliates. All rights reserved.
+ * Copyright (c) 2019-Present, Okta, Inc. and/or its affiliates. All rights reserved.
```

Comment:
check `yarn banners` script, the banners can be updated automatically.

Reply:
<img width="873" alt="Screenshot 2021-03-23 at 16 17 27" src="https://user-images.githubusercontent.com/77254185/112160892-48d27d00-8bf3-11eb-9c5e-3f107eeaf5eb.png">

Reply:
looks like `globby` is the root level deps of the repo. In the root folder try `rm -rf node_modules` -> `npm install` -> `npm run banners`. It works for me locally on master.